lib/broccoli/ember-app.js

```diff
@@ -332,18 +332,34 @@ EmberApp.env = function() {
   Provides a broccoli files concatenation filter that's configured
   properly for this application.
 
+  @deprecated
   @method concatFiles
   @param tree
   @param options
   @return
 */
-EmberApp.prototype.concatFiles = function(tree, options) {
+EmberApp.prototype.concatFiles = function(tree, options, ignoreDeprecation) {
+  this.project.ui.writeDeprecateLine('EmberApp.concatFiles() is deprecated. Please use the `broccoli-concat` module directly.', ignoreDeprecation);
```

Comment:
like [this](https://github.com/ember-cli/ember-cli/pull/5926/commits/e41d77b3dd04374d34a643ce7cd7ff381d0761b4)?